src/main/resources/db/migration/V6_10__adding_new_user_roles_for_ia_and_fpla.sql

```diff
@@ -0,0 +1,15 @@
+ALTER TABLE PRD_ENUM ADD COLUMN ENABLED VARCHAR(8) DEFAULT 'YES' NOT NULL;
+
```

Comment:
default value should be No in the alter statement

You should first do an update of the existing rows in the table and then add insert queries